packages/lib-classifier/src/components/Classifier/components/MetaTools/components/HidePreviousMarksButton/HideTranscriptionButton.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Button } from 'grommet'
+import { bool, func, object, string } from 'prop-types'
+import { SpacedText } from '@zooniverse/react-components'
+import styled, { css } from 'styled-components'
+
+const StyledButton = styled(Button)`
+  > div {
+    justify-content: start;
+    padding: 0.5em;
+  }
+
+  span {
+    ${props => props.active && css`font-weight: bold;`}
+  }
+
+  &:focus,
+  &:enabled:hover {
+    text-decoration: underline;
+  }
+`
+
+export default function HideTranscriptionButton({ checked, disabled, icon, onClick, title }) {
+  return (
+    <StyledButton
+      active={checked}
+      a11yTitle={title}
+      aria-checked={checked}
+      disabled={disabled}
+      gap='xxsmall'
+      icon={icon}
+      label={
+        <SpacedText color={{ dark: 'accent-2', light: 'neutral-2' }}>
+          {title}
+        </SpacedText>
+      }
+      onClick={onClick}
+      pad='small'
```

Comment:
This doesn't do anything with `plain` being true (see explaination: https://v2.grommet.io/button#plain). The padding is added by the use of styled-components on line 10.

The styling here is now almost identical to the `MetaToolsButton` from the component library. This is non-blocking, but if that shared component is updated to take a pad prop and set padding via styled-components, then I think we can replace this here with the `MetaToolsButton`. Perhaps this can be a minor refactor next?